.github/workflows/build.yaml

```diff
@@ -39,9 +60,28 @@ jobs:
       - name: 'Install required integration test software'
         working-directory: ./liberty-tools-intellij
         run: bash ./src/test/resources/ci/scripts/setup.sh
+
+      # Checkout and build lsp4ij only if USE_LOCAL_PLUGIN is true
+      - name: 'Checkout lsp4ij'
+        if: env.USE_LOCAL_PLUGIN == 'true'
```

Comment:
Do we need to add quotes to boolean values?

Reply:
No longer needed. Removed the quotes